budcamp/Homework3/Homework3/task2/TestRunner.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework3
+{
+    internal class TestRunner
+    {
+        private TestContainer testContainer;
+        private TestResults testResults;
```

Comment:
Please get read of this field, your `TestRunner` should not be aware of existence of `TestResults`.